lib/cmd/compile/fonts.js

```diff
@@ -0,0 +1,192 @@
+'use strict';
+const _ = require('lodash'),
+  h = require('highland'),
+  afs = require('amphora-fs'),
+  path = require('path'),
+  es = require('event-stream'),
+  gulp = require('gulp'),
+  newer = require('gulp-newer'),
+  concat = require('gulp-concat'),
+  rename = require('gulp-rename'),
+  gulpIf = require('gulp-if'),
+  cssmin = require('gulp-cssmin'),
+  sourcePath = path.join(process.cwd(), 'styleguides'),
+  destPath = path.join(process.cwd(), 'public'),
+  // these are the font weights, styles, and formats we support
+  fontWeights = {
```

Comment:
What's this list from? We should link to source (mdn document I'm assuming?)

Reply:
W3SCHOOOOOOOLS!